src/libsyncengine/update_detection/file_system_observer/computefsoperationworker.cpp

```diff
@@ -929,4 +949,10 @@ bool ComputeFSOperationWorker::pathInDeletedFolder(const SyncPath &path) {
     return false;
 }
 
+void ComputeFSOperationWorker::notifyIgnoredItem(const NodeId &nodeId, const SyncPath &path, NodeType nodeType) {
+    LOGW_SYNCPAL_INFO(_logger, L"Item (or one of its descendant) has been ignored: " << Utility::formatSyncPath(path));
```

Comment:
```suggestion
    LOGW_SYNCPAL_INFO(_logger, L"Item (or one of its descendants) has been ignored: " << Utility::formatSyncPath(path));
```

Reply:
Done in 6c66776ee849ef7120694f5f3064e21c10907145